src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
delete it from here, it is already setted in `*` selector

Reply:
ready